src/runtime/gecko/l20n-chrome-xbl.js

```diff
@@ -8,12 +8,12 @@ Components.utils.import('resource://gre/modules/L10nRegistry.jsm');
 Components.utils.import('resource://gre/modules/IntlMessageContext.jsm');
 
 const functions = {
-  OS: function() {
+  PLATFORM: function() {
     switch (Services.appinfo.OS) {
       case 'WINNT':
-        return 'win';
+        return 'windows';
       case 'Linux':
-        return 'lin';
+        return 'linux';
       case 'Darwin':
         return 'mac';
```

Comment:
Thanks for using long names for Windows and Linux.  Do you think `macos` would fit better here as well?